platforms/karaf/itests/pom.xml

```diff
@@ -52,9 +52,8 @@
 
         <dependency>
             <groupId>org.apache.karaf.itests</groupId>
-            <artifactId>itests</artifactId>
+            <artifactId>common</artifactId>
```

Comment:
Question, Do we really need to rename it?

Reply:
yes, this is a change introduced by latest Karaf version. Some common test classes were abstract in common module and can be used outside karaf.

Reply:
oh, I see. cool